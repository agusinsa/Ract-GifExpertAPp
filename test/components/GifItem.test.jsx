const { render } = require("@testing-library/react")
const { GifItem } = require("../../src/components/GifItem")


describe('Pruebas en <GifItem />', () => { 
    
    const title = 'Dragon Ball';
    const url = 'https://dragon-ball.com/goku-ssjb.jpg'

    test('Debe hacer match con el snapshot', () => { 
        const { container}=render(<GifItem title={title} url={url} />);
        expect(container).toMatchSnapshot();

     })
 })