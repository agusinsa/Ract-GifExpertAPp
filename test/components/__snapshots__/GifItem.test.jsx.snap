// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem /> Debe hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Dragon Ball"
      src="https://dragon-ball.com/goku-ssjb.jpg"
    />
    <p>
      Dragon Ball
    </p>
  </div>
</div>
`;
